Week5/app/src/main/java/com/isabelledionisius/mymovieapp/MovieDatalist.kt

```diff
@@ -0,0 +1,28 @@
+package com.isabelledionisius.mymovieapp
+
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+
+@Entity(tableName="datalist")
+
+class MovieDataList(
```

Comment:
This formatting is a bit off. It'd be better if you didn't have so much extra lines between each variable and at the end